tuf/api/_payload.py

```diff
@@ -45,6 +48,31 @@
 T = TypeVar("T", "Root", "Timestamp", "Snapshot", "Targets")
 
 
+def _hash(algo: str) -> Any:  # noqa: ANN401
```

Comment:
Python's hashlib does not make the `Hash` type public, hence I'm using `Any` and ask the linter for forgiveness.